### ISSUE TITLE
Phaser Keyboard lastChar

### ISSUE THREAD

**Issue by digitsensitive** [photonstorm/phaser-ce#132]
Hello there!

I'm having the issue, that I can not access lastChar before any key has been pressed. I always get the following error:

phaser.js:44758 Uncaught TypeError: Cannot read property 'charCode' of null

If I press a key very quick before the game starts it works because lastChar seems not to be undefined.

I guess in this part lies the error:

```
Object.defineProperty(Phaser.Keyboard.prototype, "lastChar", {

    get: function () {

        if (this.event.charCode === '32')
        {
            return '';
        }
        else
        {
            return String.fromCharCode(this.pressEvent.charCode);
        }

    }

});
```

**Comment by samme** [comment]
A workaround is to check `game.input.keyboard.pressEvent` first.

**Comment by samme** [comment]
Fixed in #135
